src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/Extentions/StringExtentions.cs

```diff
@@ -167,6 +167,66 @@ public static string GetValidContent(this string s, CharOptions charOptions, str
             return string.IsNullOrEmpty(specialChars) ? result : result.ReplaceTwoAndMoreCharsBySomeOne(specialChars);
         }
 
+        /// <summary>
+        /// Converts string to bool. When exceptionOrDefault is true and is cast error throws exception otherwise returns false.
```

Comment:
Please add extra parameter default value = false, then RaiseException default = false to be more clear